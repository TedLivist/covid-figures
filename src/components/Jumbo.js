import React from 'react';

const Jumbo = (props) => {
  const { name, cases } = props
  
  return (
    <div>
      <p>Jumbo</p>
      <p>{name}</p>
      <p>{cases}</p>
    </div>
  );
}
 
export default Jumbo;