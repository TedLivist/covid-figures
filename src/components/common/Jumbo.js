import React from 'react';

const Jumbo = (props) => {
  const { name, cases } = props
  
  return (
    <div className="jumbo d-flex">
      <div className="icon-div w-50">
        <box-icon color="#7B2C4B" size="8rem" name='world'></box-icon>
      </div>
      <div className="w-50 jumbo-title">
        <div>{name}</div>
        <div>{cases}</div>
      </div>
    </div>
  );
}
 
export default Jumbo;