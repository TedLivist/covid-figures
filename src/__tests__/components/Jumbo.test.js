import React from 'react';
import renderer from 'react-test-renderer'
import Jumbo from '../../components/common/Jumbo';

test('renders correctly', () => {
  const tree = renderer
    .create(<Jumbo />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
