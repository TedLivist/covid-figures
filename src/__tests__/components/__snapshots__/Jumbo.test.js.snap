// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="jumbo d-flex"
>
  <div
    className="icon-div w-50"
  >
    <box-icon
      color="#7B2C4B"
      name="map"
      size="8rem"
    />
  </div>
  <div
    className="w-50 jumbo-title"
  >
    <div>
      
    </div>
    <div>
      0
    </div>
  </div>
</div>
`;
